src/ts/components/subtitleoverlay.ts

```diff
@@ -36,13 +46,18 @@ export class SubtitleOverlay extends Container<ContainerConfig> {
     player.addEventHandler(player.EVENT.ON_CUE_ENTER, (event: SubtitleCueEvent) => {
       let labelToAdd = subtitleManager.cueEnter(event);
 
-      if (this.previewSubtitleActive) {
-        this.removeComponent(this.previewSubtitle);
+      if (this.isCEA608) {
```

Comment:
That should of course not happen, hence my question below :) Still it does not make sense to extract cea608 handling to a separate configure function if it is not self-contained.